qa/rpc-tests/disablewallet.py

```diff
@@ -10,6 +10,7 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import *
 
+
```

Comment:
nit: This additional newline makes this a double-newline. This doesn't seem to match the style of the other files.